src/Raven.Server/Program.cs

```diff
@@ -377,12 +377,22 @@ public static void WriteThreadsInfoAndWaitForEsc(RavenServer server, int maxTopT
             var maxNameLength = 0;
             while (Console.KeyAvailable == false)
             {
-                Console.SetCursorPosition(cursorLeft, cursorTop);
+                if (PlatformDetails.RunningOnPosix)
+                {
+                    Console.Out.Write('\x1b');
```

Comment:
What is this for?

Reply:
This tells to the terminal `set cursor position` 
http://www.tldp.org/HOWTO/Bash-Prompt-HOWTO/x361.html